README.md

```diff
@@ -47,9 +47,28 @@ Use the `.brandrc.example` file as a template: `cp .brandrc.example .brandrc` an
 * `npm run start`
 
 ## :books: Content repository structure
-Repository stucture can be altered through `.activities-rc.json` file. Use the `.activities-rc.json.example` file as a template: `cp .activities-rc.json.example .activities-rc.json` and enter configuration details. It's also possible to use `.activities-rc.js`, by using `module.exports` to export the structure.
-By default, the file is searched for in the root of the project. If a custom location or a custom name is needed, it can be provided through the `activitiesConfig` param to any of the build scripts (without the extension), for example:
-`npm run dev:client -- --activitiesConfig=server/.custom-activities-rc`
+Repository stucture can be altered using activities configuration file.
+Recongnized configuration filenames are (sorted by priority): 
+- `activities.config.js`
```

Comment:
Should we rename this to schemas or something along those lines / better ? 🤔

Reply:
With backward compatibility support through next few versions? Meaning we warn users they should rename it from `.activities-rc` to `.schemasrc` or `schemas.config.js`?